spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, :type => :controller do
+
+  let(:valid_attributes) { {title: 'Lorem', body: 'Ipsum'} }
+
+  let(:invalid_attributes) {
+    skip("Add a hash of attributes invalid for your model")
+  }
+
+  let(:valid_session) { {} }
+
+  describe "GET index" do
+    it "assigns all posts as @posts" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.